robottelo.properties.sample

```diff
@@ -60,3 +60,8 @@ admin.password=changeme
 driver=firefox
 
 # NOTE: Candlepin url accepts just the hostname.
+
+# Section for declaring Sat5->Sat6 transition parameters
+[transitions]
+# Path to exported data archive (typically a .tgz containing a bunch of CSV files together with repo data
+export_tar.url=
```

Comment:
it is a URL indeed :)

The parameter is only used by `prepare_import_data()` method in case no local path is specified. So it downloads the default.
We wanted to have some sort of 'zero config' option here